README.md

```diff
@@ -17,10 +17,12 @@ npm install warehouse.ai-status-models --save
 ## Usage
 
 ```
-const datastar = require('datastar')({ /* connection config */ }).connect();
-const models = require('warehouse.ai-status-models')(datastar);
+const dynamoClient = new AWS.DynamoDB({ /* connection config */ });
+const dynamodb = require('dynamodb');
+dynamodb.dynamoDriver(dynamoClient);
+const models = require('warehouse.ai-status-models')(dynamodb);
 
-// from datastar.define we get...
+// from dynastar.define we get...
 const Status = models.Status;
 
 Status.findFirst({ ... }, function (err, data) { .... });
```

Comment:
remove findFirst reference since we donut support it.

Reply:
mmmm donuts...

![donuts](https://media.giphy.com/media/Fcrw8XQlFlYCk/giphy.gif)